common/persistence/src/main/java/app/coronawarn/server/common/persistence/domain/DiagnosisKeyBuilders.java

```diff
@@ -45,7 +45,16 @@
      * @param protoBufObject ProtocolBuffer object associated with the temporary exposure key.
      * @return this Builder instance.
      */
-    FinalBuilder fromProtoBuf(TemporaryExposureKey protoBufObject);
+    FinalBuilder fromTemporaryExposureKey(TemporaryExposureKey protoBufObject);
+
+    /**
+     * Adds the data contained in the specified federation diagnosis key key object to this builder.
+     *
+     * @param federationDiagnosisKey DiagnosisKey object associated with the temporary exposure key.
+     * @return this Builder instance.
+     */
+    FinalBuilder fromFederationDiagnosisKey(
+        app.coronawarn.server.common.protocols.external.exposurenotification.DiagnosisKey federationDiagnosisKey);
```

Comment:
@mibrasap please add 'app.coronawarn.server.common.protocols.external.exposurenotification.DiagnosisKey' to the import and use here the short name then